common/src/main/java/org/opensearch/ml/common/MLTask.java

```diff
@@ -65,6 +75,8 @@ public class MLTask implements ToXContentObject, Writeable {
     private String error;
     private User user; // TODO: support document level access control later
     private boolean async;
+    @Setter
+    private Map<String, Object> transformJob;
```

Comment:
why not just use Map<String, String>. I think all the job id will be parsed as a String? So using parser.mapString() is easier.